ChroniquesOublieesFantasy2 Version 1.1.0

stephaned68 opened the pull request:
<!-- ATTENTION: This Pull Request template changed on 03/17/22. Please ensure that you are completing this template to the fullest extent possible. -->

# Submission Checklist
## Required

<!-- Check these off by adding an 'x' to each of these boxes. If you fail to meet all these criteria, your PR will be rejected. -->

- [x] The pull request title clearly contains the name of the sheet I am editing.
- [x] The pull request title clearly states the type of change I am submitting (New Sheet/New Feature/Bugfix/etc.).
- [x] The pull request makes changes to files in only one sub-folder.
- [x] The pull request does not contain changes to any json files in the translations folder (translation.json is permitted)

## New Sheet Details

<!-- If you are submitting a new sheet to the repository, please fill in any empty spaces indicated by < >. -->

- The name of this game is: <   >  _(i.e. Dungeons & Dragons 5th Edition, The Dresden Files RPG)_
- The publisher of this game is: <   > _(i.e. Wizards of the Coast, Evil Hat)_
- The name of this game system/family is: <   > _(i.e. Dungeons & Dragons, FATE)_

- [ ] I have followed the [Character Sheets Standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) when building this sheet.

<!-- Please check any that apply: -->

- [ ] I have authorization from the game's publisher to make this an official sheet on Roll20 with their name attached.
- [ ] This game is not a traditionally published game, but a copy of the game rules can be purchased/downloaded/found at: <   >
- [ ] This sheet is for an unofficial fan game, modification to an existing game, or a homebrew system.

# Changes / Description

- Changed png files URLs to Roll20 community sheets repo
- Added button for luck tests
- Added parameter and prop fields for each capability
- Added Path's Vigor Point for eah path
- Rank 1 of path number 1,2 & 3 checked by default
- Added an option for automatic calculation of max Vigor Points
- Added support for companions (attributes dependencies)
- Added attack options (COFantasy2 MOD script enhancements)
- Replaced JSON-based profile import with text-based path import from rulebook PDF
- Added a button to add profile's base gear

roll20deploy commented:
[Character Sheet Info](https://app.roll20.net/managesheets/ChroniquesOublieesFantasy2/roll20-character-sheets) *Roll20 Internal Use only.*

stephaned68 commented:
Hi @NorWhal 
I have changed CRLF to LF in VSCode, re-submitted to my own repo then copied the raw file to my fork of the Roll20 repo
I had already added a couple of new features to the sheet since the submission and corrected a minor bug, so the code is not the same even after changing the line-endings
I see that all checks have passed so hopefully you'll be able to merge. TIA

NorWhal commented:
You still need to change the html and css files - they still have CRLF line endings. You will know you've done it correctly when every line in the file is marked as changed in the PR's "Files Changed" tab.

stephaned68 commented:
Hi @NorWhal 
This situation is very odd as this is the first time Roll20 asks for this CRLF vs LF 
However, I think I managed to do the change you requested by using Github Desktop & VSCode to change the CRLF to LF and re-commiting to the PR
Do not bother with the commit message (`COF2E Version 2.1.0 (LF-endings, 2nd attempt)`), this is still Version 1.1.0 of the sheet, I made a typo due to annoyment and frustration
TIA

NorWhal commented:
Thank you for taking the effort, @stephaned68, it's greatly appreciated. I've just gone into the repo and checked; none of your other sheets had improper line endings, which is why this hasn't come up before. Maybe there's been a change to the coding environment you use, like a new machine, IDE, or templating language?

To explain the reasoning for this rule; it's for Unix server compatibility on our side, and not adhering to it is quite disruptive to other community authors that use github desktop, which will sometimes automatically change your files to use `LF` and cause their PRs to break the "One sheet per PR" rule.

stephaned68 commented:
Hi @NorWhal 

Thx for explaining the ins and outs and for merging the PR

I'm using the same IDE, same environment, no changes. I am always using the same routine when working with Roll20 : 
- committing to my local repo and pushing to a remote personal repo (I always have one, just in case)
- creating the branch in my roll20-sheets fork on Github
- copying the files (html, css, json and readme) from my remote repo and doing a commit in the fork, on Github again

My other sheets (ChroniquesOublieesContemporain & ChroniquesGalactiques) were copied from the original ChroniquesOubliees (Fantasy 1st Ed) created by NathaTerrien who used to work as a freelance dev for Roll20 (don't know if he's still around) and he possibly created the original repo with the correct line-endings. For this new sheet, I created the local repo myself and paid no attention to the line-endings, it may explain the discrepancy. 

Thx again. Best wishes for 2025 :)
